e2e-tests/security-context/conf/sec-context-rs0-changed.yml

```diff
@@ -5,6 +5,9 @@ metadata:
 spec:
   image:
   imagePullPolicy: Always
+  initContainerSecurityContext: 
+    privileged: false
+    runAsUser: 2002
```

Comment:
I'm a bit confused on how this works?

Reply:
This `initContainerSecurityContext` is simply applied to the init containers security context, just like `containerSecurityContext` is applied to the container security context. Is that what you are confused about, or you are confused about the test itself?